internal/workflow.go

```diff
@@ -413,8 +413,7 @@ type (
 		// inside a workflow as a child workflow.
 		Name                          string
 		DisableAlreadyRegisteredCheck bool
-		// Optional: Provides a default Versioning Behavior to workflows of this type.
-		// See workflow.SetVersioningBehavior to override this default.
+		// Optional: Provides a Versioning Behavior to workflows of this type.
```

Comment:
Should we mention that this is not optional of there is no default behavior and `DeploymentOptions.UseBuildIDForVersioning==true`?

Reply:
Good point, I'll make this more clear.